project/program/templates/program/speaker.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+{% block content %}
+<article>
+    <button class="btn btn-secondary"> < Back to speakers</button>
```

Comment:
`<` -> `&lt;`
το `<` είναι [δεσμευμένο](https://stackoverflow.com/questions/7381974/which-characters-need-to-be-escaped-in-html) στην html